.circleci/config.yml

```diff
@@ -111,20 +111,20 @@ jobs:
 
       - run:
           name: Javadoc
-          command: make ci-javadoc
+          command: make javadoc
       - store_artifacts:
-          path: ci/javadoc/build/reports
+          path: code/edge/build/dokka/javadoc
```

Comment:
Should we put into 
code/edge/build/docs/javadoc
as Userprofile?

Reply:
This is reading the generated HTML files after running Javadoc. Edge is using Dokka to generate the Javadocs so this is the path where the files are located. We may want to move away from Dokka, as it appears the other extensions using Kotlin are not using it. But if we do, that will be in another PR.